packages/adapter-cloudflare/index.js

```diff
@@ -217,6 +221,21 @@ function get_routes_json(builder, assets, { include = ['/*'], exclude = ['<all>'
 				return builder.prerendered.paths;
 			}
 
+			if (rule === '<redirects>' && existsSync(redirects_file)) {
+				const redirects = readFileSync(redirects_file, 'utf8')
```

Comment:
you might be able to put everything after reading the file into a method like `parse_redirects` that has a unit test. I get a bit nervous that the adapters keep growing without having any tests

Reply:
I've added a test but I'm not sure if it's enough. Should I add another one to test the case where it fails?